visualizations/interactiveviewer/config/interactiveviewer.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="Interactive Viewer">
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test test_attr="creating_job.tool_id">
+                quantp
+            </test>
+            <test test_attr="creating_job.tool_id">
+                toolshed.g2.bx.psu.edu/repos/galaxyp/quantp/quantp/1.1.0
```

Comment:
Wow, cool didn't know this is possible :)

Reply:
It's super useful! I learned this trick [here](url
https://github.com/galaxyproject/galaxy-hub/blob/master/src/visualizations-registry/cookbook/index.md#i-want-my-visualization-to-work-only-on-datasets-produced-by-a-specific-tool).